index.html

```diff
@@ -0,0 +1,145 @@
+                <p class="date font-face">LEARN EVERYDAY + LIFE TIME SUPPORT</p>
+                <p class="location font-face">We are fully online | Support team available 24 hours |</p>
+            </div>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="program-bg"></div>
+        <div class="p-cont">
+            <p class="program-title">Main Program</p>
+            <hr>
+        </div>
+        
+        <div class="container wrapper">
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/lecturer.png" alt="lecture image">
+                    <p class="t1">Lecture</p>
+                </div>
+                <p class="program-disc">Watch some of our meterials from various expect across the world.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/board.png" alt="lecture image">
+                    <p class="t1">Exhibition</p>
+                </div>
+                <p class="program-disc">Our board contain timetables and announcements.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/chat.png" alt="lecture image">
+                    <p class="t1">Forum</p>
+                </div>
+                <p class="program-disc">Have questions, drop us as chat.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/workshop.png" alt="lecture image">
+                    <p class="t1">Mentors</p>
+                </div>
+                <p class="program-disc">You are assigned a personal mentor</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/meeting.png" alt="lecture image">
+                    <p class="t1">Meetings</p>
+                </div>
+                <p class="program-disc">We have weekly meetings to address challenges we are faced.</p>
+            </div>
+        </div>
+        <div class="button">
+            <a class="start-learning" href="#">Start Learning</a>
+        </div>
+        <a class="whole-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+     <!-- speaker section -->
+     <section class="container main-speakers">
+        <div class="p-cont">
+            <p class="program-title">Featured Speakers</p>
+            <hr>
+        </div>
+
+        <div id="dynamic-page" class="row"></div>
+        <div id="more" class="row grid"></div>
+      
+        <div class="read-button">
+            <button onclick="ReadMore()" class="read-more" id="toggle">MORE<i class="fa fa-angle-down fa-lg"></i></button>
+        </div>
+    </section>
```

Comment:
- [ ] As suggested by the previous reviewer, there is overlapping of elements here. On mobile, your texts overlapped the images and as a result, it has distorted the fine outlook that section is supposed to have got. Kindly look into it and fix it.

### Your Design

![your-speaker-section](https://user-images.githubusercontent.com/26597432/197186042-ee5bd4f9-e885-4d6d-8466-ae2783217b22.png)


### Template

![temp-speaker](https://user-images.githubusercontent.com/26597432/197186084-d11d24ea-f1da-4f77-badd-17de9bde46e0.png)